core-chart/templates/NOTES.txt

```diff
@@ -1,53 +1,54 @@
-Wait for all Control Planes to be ready:
+Below is a code snippet that allows to wait for all KubeFlex Control Planes to be ready:
 
-echo "Waiting for all Control Planes to be ready..."
+echo "Waiting for all KubeFlex Control Planes to be Ready:"
 for cpname in `kubectl get controlplane -o name`; do
-    cpname=${cpname##*/}
-    while [[ $(kubectl get cp $cpname -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; do
-        echo "waiting for Control Plane $cpname..."
-        sleep 5
-    done
-    echo $cpname
+  cpname=${cpname##*/}
+  while [[ `kubectl get cp $cpname -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}'` != "True" ]]; do
+    echo "Waiting for \"$cpname\"..."
+    sleep 5
+  done
+  echo "\"$cpname\" is ready."
 done
 
-Control Planes of type host can be accessed using the host cluster kubeconfig.
+While KubeFlex Control Planes of type host can be accessed using the host cluster kubeconfig, the following
+code snippet can be used to obtain the kubeconfig to access a Control Plane $cpname of another type:
 
-Obtain the kubeconfig to access a Control Plane $cpname of another type with the command:
-
-echo "Getting the kubeconfig of Control Planes $cpname ==> ./kubeconfig-$cpname..."
+echo "Getting the kubeconfig of KubeFlex Control Planes \"$cpname\" ==> \"./kubeconfig-$cpname\"..."
 kubectl get secret $(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.name}') \
-    -n $(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.namespace}') \
-    -o=jsonpath="{.data.$(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.key}')}" \
-    | base64 -d > "./kubeconfig-$cpname"
+  -n $(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.namespace}') \
+  -o=jsonpath="{.data.$(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.key}')}" \
+  | base64 -d > "./kubeconfig-$cpname"
 
 which will create a file "kubeconfig-$cpname" in the current folder.
 
-Obtain the kubeconfig to access all Control Plane with the command:
+The following code snippet allows one to obtain the kubeconfig of all KubeFlex Control Plane in the cluster:

```

Comment:
More specifcally than "code snippet", it is some shell commands. Which shells does do they work in?